installer.js

```diff
@@ -90,15 +90,15 @@ function driverVersionString(driverName, targetPath)
     }
     else
     {
-        throw new Error(`No driver exists with the name ${driverName}.`);
+        throw new Error(`no driver exists with the name ${driverName}.`);
     }
 }
 
 function driverInstaller(browserName, browserVersion, targetPath)
 {
     if (typeof browserName !== 'string' || typeof browserVersion !== 'string' || typeof targetPath !== 'string')
     {
-        throw new Error('Parameters are not valid strings.');
+        throw new Error('parameters are not valid strings.');
```

Comment:
the <blah> parameters are not valid strings

remove the period

Reply:
fixed